amundsen_application/api/utils/request_utils.py

```diff
@@ -1,9 +1,36 @@
 from typing import Dict
-
+import requests
+from flask import Response
 
 def get_query_param(args: Dict, param: str, error_msg: str = None) -> str:
     value = args.get(param)
     if value is None:
         msg = 'A {0} parameter must be provided'.format(param) if error_msg is not None else error_msg
         raise Exception(msg)
     return value
+
+
+def make_request_wrapper(method: str, url: str, client: any, headers: any, timeout: int) -> Response:
```

Comment:
Got any suggestions for a different name?

Reply:
Maybe just `request_wrapper` if we're going for nouns...maybe `make_request` or `get_response` if we're going for a verb? No strong attachments there though.